network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/typeform_block.html

```diff
@@ -1,5 +1,7 @@
-<script src="https://embed.typeform.com/embed.js" ></script>
+<div
+  class="typeform-widget"
+  data-url="https://form.typeform.com/to/{{ value.embed_id }}?typeform-medium=embed-snippet"
+  style="width: 100%; height: {{ value.height }}px;">
+</div>
 
-<meta name="typeform-formurl" content="{{ value.url }}">
-
-<button id="btn-typeform-popup" class="btn {{ value.button_type }}">{{ value.button_text }}</button>
```

Comment:
From [embed-typeform.js](https://github.com/mozilla/foundation.mozilla.org/blob/master/source/js/embed-typeform.js) it seems like the button was used to launch the form in a new tab. Have we checked with design team to see if it's okay to remove this functionality? If so, let's clean up `embed-typeform.js` a bit.

Reply:
I will turn this into an `<a>` instead, but with button styling, so we don't actually need any JS for this to work.